web/skins/classic/views/event.php

```diff
@@ -113,7 +113,7 @@
         <span id="dataFrames" title="<?php echo translate('AttrFrames')."/".translate('AttrAlarmFrames') ?>"><?php echo $Event->Frames() ?>/<?php echo $Event->AlarmFrames() ?></span>
         <span id="dataScore" title="<?php echo translate('AttrTotalScore')."/".translate('AttrAvgScore')."/".translate('AttrMaxScore') ?>"><?php echo $Event->TotScore() ?>/<?php echo $Event->AvgScore() ?>/<?php echo $Event->MaxScore() ?></span>
         <span id="Storage"> <?php echo human_filesize($Event->DiskSpace(null)) . ' on ' . $Event->Storage()->Name() ?></span>
-        <div id="closeWindow"><a href="#" onclick="<?php echo $popup ?  'window.close()' : 'window.history.back()' ?>"><?php echo $popup ? translate('Close') : translate('Back') ?></a></div>
+        <div id="closeWindow"><a href="#" onclick="<?php echo !$popup ? 'window.history.back()' : 'window.close()' ?>"><?php echo !$popup ? translate('Back') : translate('Close') ?></a></div>
```

Comment:
minor detail but the previous version is equivalent and uses 1 less character...